src/StarRating.js

```diff
@@ -29,19 +29,19 @@ export default function StarRating({
   maxRating = 5,
   color = "#fcc419",
   size = 48,
-  className ='',
+  className = '',
   messeges = [],
   defaultRating = 0,
-  onSetRating = () =>{}
+  onSetRating = () => { }
 }) {
   const [rating, setRating] = useState(defaultRating);
   const [tempRating, setTempRating] = useState(defaultRating);
 
   const textStyle = {
     lineHeight: "1",
     margin: "0",
-    color:color,  // when that kind of case become of assigning value we can directly write 'color' only 👍
-    fontSize :`${size/1.5}px`,
+    color: color,  // when that kind of case become of assigning value we can directly write 'color' only 👍
```

Comment:
Delete this comment from here. Move it to the top of the object.

Reply:
Probably we not need to do that.

Reply:
Okay no problem